docs/en/ingest-management/elastic-agent/configuration/outputs/output-elasticsearch.asciidoc

```diff
@@ -0,0 +1,557 @@
+
+// =============================================================================
+
+include::output-shared-settings.asciidoc[tag=enabled-setting]
+
+*Default:* `true`
+
+// =============================================================================
+
+// tag::hosts-setting[]
+|
+[id="output-{output-type}-hosts-setting"]
+`hosts`
+
+| (list) The list of {es} nodes to connect to. The events are distributed to
+these nodes in round robin order. If one node becomes unreachable, the event is
+automatically sent to another node. Each {es} node can be defined as a `URL` or
+`IP:PORT`. For example: `http://192.15.3.2`, `https://es.found.io:9230` or
+`192.24.3.2:9300`. If no port is specified, `9200` is used.
+
+NOTE: When a node is defined as an `IP:PORT`, the _scheme_ and _path_ are taken
+from the `protocol` and `path` settings. 
+
+[source,yaml]
+------------------------------------------------------------------------------
+outputs:
+  default:
+    type: elasticsearch
+    hosts: ["10.45.3.2:9220", "10.45.3.1:9230"] <1>
+    protocol: https
+    path: /elasticsearch
+------------------------------------------------------------------------------
+<1> In this example, the {es} nodes are available at
+`https://10.45.3.2:9220/elasticsearch` and
+`https://10.45.3.1:9230/elasticsearch`.
+// end::hosts-setting[]
+
+// =============================================================================
+
+// tag::protocol-setting[]
+|
+[id="output-{output-type}-protocol-setting"]
+`protocol`
+
+| (string) The name of the protocol {es} is reachable on. The options are:
+`http` or `https`. The default is `http`. However, if you specify a URL for
+`hosts`, the value of `protocol` is overridden by whatever scheme you specify in
+the URL.
+// end::protocol-setting[]
+
+// =============================================================================
+
+// tag::proxy_url-setting[]
+|
+[id="output-{output-type}-proxy_url-setting"]
+`proxy_url`
+
+| (string) The URL of the proxy to use when connecting to the {es} servers. The
+value may be either a complete URL or a `host[:port]`, in which case the `http`
+scheme is assumed. If a value is not specified through the configuration file
+then proxy environment variables are used. See the
+https://golang.org/pkg/net/http/#ProxyFromEnvironment[Go documentation]
+for more information about the environment variables.
+// end::proxy_url-setting[]
+
+// =============================================================================
+
+|===
+
+[[output-elasticsearch-authentication-settings]]
+== Authentication settings
+
+Settings for authenticating with {es}.
+
+When sending data to a secured cluster through the `elasticsearch`
+output, {agent} can use any of the following authentication methods:
+
+* Basic authentication credentials (username and password).
+* Token-based (API key) authentication.
+* Public Key Infrastructure (PKI) certificates.
+
+//QUESTION: Why isn't kerberos covered here?
```

Comment:
Good question! There is a `kerberos` field in the elasticsearch output configuration with all the usual fields, and I think it works as usual in agent. I'll try to double check there aren't any catches, but as far as I can tell right now `ssl` and `kerberos` can be included here with the same fields and defaults as documented in the beats outputs.